onnxruntime/core/providers/cpu/sequence/sequence_ops.cc

```diff
@@ -24,6 +27,7 @@ Status SequenceLength::Compute(OpKernelContext* context) const {
   ORT_ENFORCE(X != nullptr, "Got nullptr for sequence input.");
 
   auto* Y = context->Output(0, {});
+  ORT_ENFORCE(Y != nullptr, "SequenceLength: Got nullptr for output tensor");
   auto* Y_data = Y->template MutableData<int64_t>();
```

Comment:
Doesn't the graph load enforce that required inputs and outputs exist?

Reply:
Not sure. If it indeed does, I can remove these; they’re definitely an annoyance to write. A lot of the op code seems to be doing this.

Reply:
I believe schema.cc in the onnx defs directory checks required inputs and outputs in OpSchema::Verify. I tested quickly with an existing unit tests and not providing a required input failed in that check.

Reply:
I see. Thanks for checking. Ok, will keep in mind. Will not change now given the amount of time it takes to get a successful build.